tests/ctl/api/test_seed.py

```diff
@@ -436,3 +444,193 @@ async def test_load_orginizations(loguru_caplog, async_session, monkeypatch):
+            datasets = (await async_session.execute(select(Dataset))).scalars().all()
+            policies = (await async_session.execute(select(PolicyCtl))).scalars().all()
+            connections = (
+                (await async_session.execute(select(ConnectionConfig))).scalars().all()
+            )
+            dataset_configs = (
+                (await async_session.execute(select(DatasetConfig))).scalars().all()
+            )
+            assert len(systems) == 4
+            assert len(datasets) == 3
+            assert len(policies) == 1
+            assert len(connections) == 2
+            assert len(dataset_configs) == 2
+
+            assert sorted([e.fides_key for e in systems]) == [
+                "cookie_house",
+                "cookie_house_marketing",
+                "cookie_house_mongo",
+                "cookie_house_postgres",
+            ]
+            assert sorted([e.fides_key for e in datasets]) == [
+                "mongo_test",
+                "postgres_example_test_dataset",
+                "stripe_connector",
+            ]
+            assert sorted([e.fides_key for e in policies]) == ["sample_policy"]
+
+            # NOTE: Only the connections configured by SAMPLE_ENV_VARS above are
+            # expected to exist; the others defined in the sample_connections.yml
+            # will be ignored since they are missing secrets!
+            assert sorted([e.key for e in connections]) == [
+                "postgres_connector",
+                "stripe_connector",
+            ]
+            assert sorted([e.fides_key for e in dataset_configs]) == [
+                "postgres_example_test_dataset",
+                "stripe_connector",
+            ]
+
+    async def test_load_sample_resources_strict(self):
+        """
+        Ensure that the resource files in the sample project are all
+        successfully parsed by the load_sample_resources_from_project()
+        function. This makes sure we don't make some changes to the sample
+        project files that aren't automatically loaded into the database via
+        this function.
+
+        NOTE: If you've found this test, that probably means you were making
+        some changes to the code and this failed unexpectedly. Maybe you removed
+        a field, changed a default, or wanted to edit some sample data?
+
+        To fix the test, you just need to ensure the code has all the logic it
+        needs to parse everything from this directory:
+        - src/fides/data/sample_project/sample_resources/*.yml
+
+        See src/fides/api/ctl/database/samples.py for details.
+
+        Sorry for the trouble, but we want to ensure there isn't a subtle bug
+        sneaking into our sample project code!
+        """
+        error_message = (
+            "Unexpected error loading sample resources; did you make changes to the sample project? "
+            "See tests/ctl/api/test_seed.py for details."
+        )
+        try:
```

Comment:
The `try` and the `assertion` here aren't really logically connected to me. As it is currently written, it seems as if it is actually checking the validity of the `resources_dict`

I recommend something like this:
```python
try:
    samples.load_sample_resource_from_project(strict=true)
    assert True
except Exception as exc:
     print(exc)
    assert False
```

To me this makes it more clear that if the command doesn't fail, it's considered passing

Reply:
That makes sense, will tweak!

Reply:
Done